apps/consumer-client/src/pages/examples/gpc-proof.tsx

```diff
@@ -195,7 +200,7 @@ export function openGPCPopup(
       value: proofConfig,
       userProvided: false
     },
-    pod: {
+    [`${PODPCD_ARG_PREFIX}_pod0`]: {
```

Comment:
There needs to be some way of exercising multiple PODs from consumer-client to validate this, particularly the UI aspects.